cupsfilters/raster.c

```diff
@@ -1571,7 +1571,7 @@ raster_base_header(cups_page_header_t *h,  // O - Raster header
     {
       if (*(val + 3) == '_' || *(val + 3) == '-')
 	ptr = val + 4;
-      colorspace = 1;
+      colorspace = pwg_raster ? 19 : 1;
       numcolors = 3;
     }
```

Comment:
The color mode setting for "Gray" (line 1549 - 1555) should be done accordingly:
```
else if (!strncasecmp(val, "Gray", 4))
    {
      if (*(val + 4) == '_' || *(val + 4) == '-')
	ptr = val + 5;
      colorspace = pwg_raster ? 18 : 3;
      numcolors = 1;
    }
```

Reply:
Pushed the changes, thank you for the review!